# Question

Is this function safe? Why or why not?

```js
const getUser = async (userId) => {
  const url = `https://jsonplaceholder.typicode.com/users/${userId}`;
  const response = await fetch(url);
  const users = await response.json();
  console.log(users)
  return users;
}
getUser(-1)
```

Again, running the code gives you the answer, so try to figure it out on your own first!

# Answer
I believe that this function is not safe because, their is no error handler, so when the function is called with negative one nothing will happen.

# Teacher Feedback

Not quite? What is missing from this code? Is this fetch call working? Should it work? 

Score: 1/3